openapi-v1.yaml

```diff
@@ -5489,6 +5593,146 @@ paths:
+    parameters:
+      - name: organization
+        in: path
+        description: organization name
+        required: true
+        type: string
+    get:
+      tags:
+        - organization
+      description: >
+        Lists all the SSO connections associated with the given organization.
+      operationId: getSSODomains
+      responses:
+        200:
+          description: The SSO domains associated with the org.
+          schema:
+            $ref: '#/definitions/SSODomainConfigResponse'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+
+  /organizations/{organization}/sso_domains/{uuid}:
+    parameters:
+      - name: organization
+        in: path
+        description: organization name
+        required: true
+        type: string
+      - name: uuid
+        in: path
+        description: configuration ID
+        required: true
+        type: string
+    get:
+      tags:
+        - organization
+      description: Gets details about the given SSO domain connection.
+      operationId: getSSODomain
+      responses:
+        200:
+          description: The details about this domain connection.
+          schema:
+            $ref: '#/definitions/SSODomainConfig'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+    patch:
+      tags:
+        - organization
+      description: Updates the configuration for the given SSO connection.
+      parameters:
+        - name: config
+          in: body
+          description: The changes to make to the configuration.
+          schema:
+            $ref: '#/definitions/SSODomainConfig'
+          required: true
+      operationId: updateSSODomain
+      responses:
+        200:
+          description: The updated details about the SSO connection.
+          schema:
+            $ref: '#/definitions/SSODomainConfig'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+    delete:
+      tags:
+        - organization
+      description: Deletes the configuration for the given SSO connection.
+      operationId: deleteSSODomain
+      responses:
+        204:
+          description: Deletion successful.
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+
+  /organizations/{organization}/sso_domains/{uuid}/run_check:
+    parameters:
+      - name: organization
+        in: path
+        description: organization name
+        required: true
+        type: string
+      - name: uuid
+        in: path
+        description: configuration ID
+        required: true
+        type: string
+    post:
+      tags:
+        - organization
+      description: >
+        Immediately verify ownership of the specified SSO domain ownership
+        claim.
+      operationId: checkSSODomain
+      responses:
+        200:
```

Comment:
We are missing the error for the `run_check`

```
        default:
          description: error response
          schema:
            $ref: '#/definitions/Error'
```

Reply:
haha holy shit I forgot about this change for literal months. fixed.